sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosFullTextPolicy.java

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Full Text Search Policy
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CosmosFullTextPolicy {
+    @JsonProperty(Constants.Properties.DEFAULT_LANGUAGE)
+    private String defaultLanguage;
+    @JsonProperty(Constants.Properties.FULL_TEXT_PATHS)
+    private List<CosmosFullTextPath> paths;
+//    private JsonSerializable jsonSerializable;
```

Comment:
We should remove it if its not required anymore.

Reply:
Updated